src/dash-table/derived/table/tooltip.ts

```diff
@@ -24,30 +29,42 @@ function getSelectedTooltip(
         return undefined;
     }
 
-    const {id, row} = currentTooltip;
+    const {header, id, row} = currentTooltip;
+
+    const tooltipSource = header ? tooltip_header : tooltip_data;
 
     if (id === undefined || row === undefined) {
         return undefined;
     }
 
+    const staticTooltip = tooltip_static?.[id];
+    const staticUseWith =
+        staticTooltip && typeof staticTooltip !== 'string'
+            ? staticTooltip.use_with
+            : TooltipUsage.Both;
```

Comment:
Apply `tooltips` to both data and headers by default. This is a change from the pre-existing behavior that would have shown the tooltip only on the data cells but it seems consistent in spirit with the new functionality.